types/rbactypes.go

```diff
@@ -23,6 +23,7 @@ type User struct {
 	UserGroups   []string
 	UserRoles    []Role
 	UserBindings []Binding
+	Properties   map[string]any
```

Comment:
Are we adding this new feature in the PR? This is an important feature; please change the PR title and description to reflect this change so that it gets its proper space in the release (or move it to another PR altogether)

Reply:
It's not a new feature, it's a cleanup. Before, properties are handled only in CreateRegoInput https://github.com/rond-authz/rond/blob/829a3dd25588aaf91038e7cea1ab9de4e41d8830/core/opaevaluator.go#L372-L376.

But this information is related to the user, as userId and groups which are already taken in the `RetrieveUserBindingsAndRoles` function.

Reply:
They weren't put in the rego input, though... Am I missing something? 🤔

Reply:
This is the entire actual function https://github.com/rond-authz/rond/blob/829a3dd25588aaf91038e7cea1ab9de4e41d8830/core/opaevaluator.go#L368-L433

At line 407, permissions are set in the input user